provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
+}
```

Comment:
Dopln prosim `default_tags` https://github.com/keboola/sandboxes-service/blob/8406d9dc004904dd374f2c6f3337f93945fe3358/provisioning/local/aws.tf#L5-L9

Reply:
To u libky nema zadny vyznam kdyz to neni aplikace ne? Proto jsem to sem nedaval.

Reply:
Vyreseno, uz jsem pridal tagy